jekyll/_cci2/circleci-images.md

```diff
@@ -93,37 +106,93 @@ The following packages are installed with `apt-get` on every image:
 - [xvfb](https://packages.debian.org/stretch/xvfb)
 - [zip](https://packages.debian.org/stretch/zip)
 
-### Other Packages
-
 The following packages are installed via `curl` or other means:
 
 - [Docker client](https://docs.docker.com/install/)
 - [Docker Compose](https://docs.docker.com/compose/overview/)
 - [dockerize](https://github.com/jwilder/dockerize)
 - [jq](https://stedolan.github.io/jq/)
 
-## Image Variants
+## Customizing a Convenience Image
+
+You can customize convenience images by adding image tags.
+There are three good reasons to do this:
+
+- You want to pin an image to a certain version or operating system (OS).
+- You want to specify a particular instance of an image with a SHA.
+- You want to modify an image by installing additional tools
+or changing the behavior of the image.
 
-CircleCI maintains variants of convenience images. These can be created by adding optional suffixes to the end of image tags.
+### Pinning an Image Version or OS
 
-**Note:** If you choose to use the `latest` tag, the image may change unexpectedly and create surprising results.
+Since convenience images are based on the **latest** versions of upstream images,
```

Comment:
We might want to be careful with using the term `latest` here. `latest` can be a Docker tag, but we don't just use that tag from upstream. Instead, we base our images off of the most recent built Docker image tags fro upstream.